src/compat/array/sortedIndexOf.spec.ts

```diff
@@ -0,0 +1,148 @@
+import { describe, expect, it } from 'vitest';
+import { sortedIndexOf } from './sortedIndexOf';
+
+describe('sortedIndexOf', () => {
+  //--------------- Lodash Test Case #1 -------------------
+  it('should find first occurrence in numeric array', () => {
```

Comment:
```suggestion
  it('should perform a binary search', () => {
```
I think it would be better to match the test title with Lodash. 
```ts
it(`\`_.${methodName}\` should accept a falsey \`array\``, () => {
                const expected = lodashStable.map(falsey, lodashStable.constant(-1));

                const actual = lodashStable.map(falsey, (array, index) => {
                    try {
                        return index ? func(array) : func();
                    } catch (e) {}
                });

                expect(actual).toEqual(expected);
            });
```
Also, it seems like the above test case is missing. Could you check on that?